### ISSUE TITLE
navDoc Landmarks

### ISSUE THREAD

**Issue by nekennedy** [w3c/publ-cg#65]
- Which landmarks drive reading system functionality (bodymatter, TOC, ect)
- What is that functionality, is it consistent across reading systems
- What other landmarks should be provided for accessibility (this should probably just be a link to accessibility best practices)
- Should there be a heading, and if so what should it be?
- Which landmarks should map to which guide items? (not sure if this should be a recommendation we make, but I think given the current retailer landscape it's probably still a best practice for backwards compatability)
